ansible/deploy-bigbluebutton.yml

```diff
@@ -0,0 +1,11 @@
+---
+
+- import_playbook: ansible-version.yml
+
+- name: "Deploy a BigBlueButton server"
+  hosts: "{{ target if target is defined else 'bigbluebutton' }}"
+  become: true
+
+  roles:
+    - role: bigbluebutton
+      tags: molecule-idempotence-notest
```

Comment:
What does this tag do? I would assume it disables the idempotence molecule test as you referred to as having issues, but I don't see this tag being referenced anywhere. How does it work?

Reply:
It is actually instruct molecule to **not check** for idempotency on this specific role.
We could add an issue to look into it and make idempotency work for this role.

Reply:
I think it's fine. It's just that I didn't see any code referencing this tag. Can you point me to the line where molecule does this? Or is it in the other commit?

Reply:
https://molecule.readthedocs.io/en/latest/configuration.html#id7

*Molecule will skip tasks which are tagged with either molecule-notest or notest. With the tag molecule-idempotence-notest tasks are only skipped during the idempotence action step.*

Reply:
thanks